doc/how_to_guides/usage.md

```diff
@@ -33,64 +34,83 @@ Runtime options:
 ```bash
 bundle exec rails generate onesie:task MyTask      # creates a normal priority task
 bundle exec rails generate onesie:task MyTask high # creates a high priority task
+
 ```
 
 ### Rake
+The following commands should occur within a v2 shell
+(`dcc shell benchprep-v2`). For more information on setting up
+containers, please consult our
+[container setup guide](../infrastructure/how_to_guides/dev_containers/setup_guide.md).
+
+You may notice that some of these examples include an optional
+template name as their last argument. These meta-templates are
+created by developers here at BenchPrep to further facilitate the
+creation of common types of onesies. For more information on finding,
+using, and contributing onesie templates, please refer to our
+[templates](/templates.md) documentation.
 
 #### API
 ```bash
-rake onesie:new[name,priority,template]         # Generates a new Onesie Task
+rake onesie:new[name,priority,template]         # Generates a new onesie task
+
 ```
 
 #### Examples
 
 ```bash
-bundle exec rake onesie:new['MyTask']        # creates a normal priority task
-bundle exec rake onesie:new['MyTask','high'] # creates a high priority task
-bundle exec rake onesie:new['MyTask','high','SampleTemplate'] # creates a high priority task using 'SampleTemplate'
+bundle exec rake onesie:new['MyTask']                   # creates a normal priority task
+bundle exec rake onesie:new['MyTask','high']            # creates a high priority task
+bundle exec rake onesie:new['MyTask',,'SampleTemplate'] # creates a normal priority task using 'SampleTemplate'
+
 ```
 
-## Write your Onesie Task
-The generator will create a onesie template file. The task must include the
-`#run` method. Add your code to the new task file.
+## Write your onesie task
+The generator will create a onesie file for you to adjust and
+populate as needed. Your edits should occur mainly (if not
+exclusively) within the required `#run` method.
 
-For details about the Task class and the class macros, see the
+For details about the task class and the class macros, see the
 [Task](../explanations/task.md) document.
 
-## Run the Onesie Task
-There are three Rake tasks to assist with running onesies.
+## Run the onesie task
```

Comment:
I'm a litte confused about the de-capitalization of `task`. First, in headers like this, I'd expect title casing. Second, a onesie `Task`, is a formal concept with a proper name, IMO, it's not simply a "task" like a chore to do around the house. 

I would say the same for `Onesie`. It's a proper name for a formalized Benchprep concept.

Reply:
I definitely agree with you on capitalizing special use of terms. Existing usage was split so I went with the more common in both cases (headers and these terms specifically).

Reply:
I've now made an attempt to make everything consistent the other way (capitalize headers and special use of words).